README.md

```diff
@@ -1,90 +1,84 @@
 # puppetlabs-self_service
 
-
-1. [Description](#description)
-1. [Setup - The basics of getting started with self_service](#setup)
-    * [What self_service affects](#what-self_service-affects)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with self_service](#beginning-with-self_service)
-1. [Usage - Configuration options and additional functionality](#usage)
-1. [Reference - what to do when a indicator repors a fault](#reference)
-1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Development - Guide for contributing to the module](#development)
+- [Description](#description)
+- [Setup](#setup)
+  - [What self_service affects](#what-self_service-affects)
+  - [Setup Requirements](#setup-requirements)
+  - [Beginning with self_service](#beginning-with-self_service)
+- [Usage](#usage)
+  - [Class Declaration *Optional.*](#class-declaration-optional)
+- [Reference](#reference)
+- [Limitations](#limitations)
+- [Development](#development)
 
 ## Description
 
-puppetlabs-self_service aims to provide a mechnism to alert the end-user when Puppet Enterprise is not in an ideal state.
+puppetlabs-self_service aims to provide a mechanism to alert the end-user when Puppet Enterprise is not in an ideal state.
 It uses pre-set indicators and has a simplified output that directs the end-user to next steps for resolution.
 
 Users of the tool should expect greater ability to provide the self served resolutions, as well as shorter incident resolution times with Puppet Enterprise Support due to higher quality information available to support cases.
 
-
 ## Setup
 
 ### What self_service affects
 
-This module installs a structured fact named self_service, which contains an array of key pairs tha simply output an indicator ID and a boolean value. 
+This module installs a structured fact named self_service, which contains an array of key pairs tha simply output an indicator ID and a boolean value.
```

Comment:
This module installs a structured fact named self_service, which contains an array of key pairs **that** output an indicator ID and a boolean value.

Reply:
Done